tests/test_brighterFatter.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of ci_cpp_gen2.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import unittest
+import lsst.utils.tests
+
+
+# TODO: DM-26396
+#       Update these tests to validate calibration construction.
+class BrighterFatterTestCases(lsst.utils.tests.TestCase):
+
+    def test_canary(self):
+        """Test to see if the brighter fatter kernel has changed.
+
+        Notes
+        -----
+        DMTN-101 14.1
+        "Characterize".
+        """
+        pass
```

Comment:
How are you planning to do this, out of curiosity? Hard code a kernel in here, and just assert they're the same?

Reply:
However though, maybe worth adding a TODO: and a ticket as, I know if this were me, I'd forget to come back and make that.